lib/assets/debug_pipeline/main.css

```
@@ -0,0 +1,203 @@
+.app {
+  font-family: "JetBrains Mono", monospace;
+  color: var(--gray-700);
+
+  --gray-50: #f8fafc;
+  --gray-100: #f0f5f9;
+  --gray-200: #e1e8f0;
+  --gray-300: #cad5e0;
+  --gray-400: #91a4b7;
+  --gray-500: #61758a;
+  --gray-600: #445668;
+  --gray-700: #304254;
+  --gray-800: #1c2a3a;
+  --gray-900: #0d1829;
+
+  --blue-600: #3e64ff;
+
+  --red-600: #db1920;
+
+  --yellow-bright-200: #fef08a;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 12px;
```

With the smaller font, the original 8px are the correct value for equal spacing on both top and bottom. :)

```suggestion
  margin-bottom: 8px;
```